profiles/arch/powerpc/ppc64/64le/package.mask

```diff
@@ -73,25 +60,10 @@
 -sys-kernel/vanilla-kernel
 -virtual/dist-kernel
 
-# Georgy Yakovlev <gyakovlev@gentoo.org> (2020-01-25)
-# libva unmasked on little-endian profile
--net-im/telegram-desktop
-
-# Georgy Yakovlev <gyakovlev@gentoo.org> (2020-01-21)
-# works fine on little-endian
--media-libs/libva
--media-video/libva-utils
--media-plugins/gst-plugins-vaapi
-
 # Georgy Yakovlev <gyakovlev@gentoo.org> (2019-08-14)
 # works fine on little-endian
 -x11-drivers/xf86-video-amdgpu
 
-# Georgy Yakovlev <gyakovlev@gentoo.org> (2019-02-04)
-# Temurin ppc64 build is little-endian only
-# so we can use bins.
--dev-java/openjdk-bin
```

Comment:
why? We unmask openjdk-bin since we have official binary for this ppc64le?

Reply:
this was reported as UnmatchedProfilePackageUnmask, so I assumed it was free to cleanup.